File: BE_App_bus/Data_Departure.php
<?php

require_once('koneksi.php');

$query = "SELECT * FROM departure ORDER BY id_departure DESC";


$sql = mysqli_query($database, $query);

if($sql){
$result = array();
    while($row = mysqli_fetch_array($sql)){
        array_push($result, array(
                'id_departure' => $row['id_departure'],
                'destination' => $row['destination'],
                'name_bus' => $row['name_bus'],
                'date' => $row['date'],
                'time' => $row['time'],
                'price' => $row['price']

        ));
    }

    echo json_encode(array('DerpatureBus'=>$result));
}


?>

File: BusTicket/app/src/main/java/com/about/busticket/Model/Departure/DataDeparture.kt
package com.about.busticket.Model.Departure

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class DataDeparture(

    var idDeparture : String ?,
    var destination : String ?,
    var name_bus : String ?,
    var date : String ?,
    var time : String ?,
    var price : String ? ):Parcelable